internal/view/reference.go

```diff
@@ -40,7 +40,7 @@ func (r *Reference) Init(ctx context.Context) error {
 
 func (r *Reference) bindKeys(aa ui.KeyActions) {
 	aa.Delete(ui.KeyShiftA, tcell.KeyCtrlS, tcell.KeyCtrlSpace, ui.KeySpace)
-	aa.Delete(tcell.KeyCtrlW, tcell.KeyCtrlL, tcell.KeyCtrlZ)
+	aa.Delete(tcell.KeyCtrlW, tcell.KeyCtrlL)
```

Comment:
This should now be KeyCtrlT since we renamed it ;(

Reply:
Added KeyCtrlT back